dist/client.conf

```diff
@@ -0,0 +1,5 @@
+# fedora-coreos-metrics client configuration
+
+[metrics]
+# Default metrics.level is `minimal`. May be set to `minimal`, `full`, or `off`.
+level=minimal
```

Comment:
I think you may want to split this knob into two: a "collecting" section with "level" (minimal/full/etc), and a "reporting" section with "enabled" (bool).

Reply:
:+1: sounds better to separate them - added this change!